.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Dependent Issues
+
+on:
```

Comment:
This job is going to run *a lot*. Every time an issue or PR is changed at all, and every hour. Do we really think this ability to mark dependencies is important enough to justify starting a container and running these scans likely at least order of 10k, likely order of 100k, times per year?

Reply:
This job will run on those PRs that have a "Depends on/ depends on" in comment section and not on all PRs. It not only marks the dependencies, it lets you about the dependency so that a PR is not merged before its dependent PR is merged.

Reply:
It's probably very short lived, let's see how it goest.

Reply:
It has to scan every issue and PR at least.

Reply:
I think Janki modified the Issue/PR event triggers to only run on open/re-open, which would reduce the number of runs dramatically. That seems like a decent compromise to me.

Reply:
That would make sense, may be then also daily?

I guess the periodic is just a fall back to fix any lost event due to CI issues.

Reply:
I think daily is good, especially while we're kinda piloting this.

Reply:
Yes, I modified the issue/PR trigger. But I feel daily is still too long a time gap. However, I want to get this merged ASAP. So going with daily checks for now. We can always edit it later.